Added German translation (#13)
TheThingsNetwork/Manifest: README.de.md

# Das „The Things Network“ Manifest

Alles, was mit elektrischer Energie betrieben wird, wird auch irgendwann mit dem Internet verbunden werden.

> Diejenigen, die dieses Netzwerk kontrollieren, werden die Welt kontrollieren. *__Wir glauben, dass eine solche Macht nicht auf wenige Personen, Firmen oder Nationen verteilt sein sollte. Wir glauben eine solche Macht sollte über so viele Personen wie möglich verteilt werden.__* Um das zu erreichen haben wir das „The Things Network“ gegründet.

Reviewer: ```suggestion
Diejenigen, die dieses Netzwerk kontrollieren, werden die Welt kontrollieren. *__Wir glauben, dass eine solche Macht nicht auf wenige Personen, Firmen oder Nationen konzentriert sein sollte. Wir glauben, dass eine solche Macht an so viele Personen wie möglich verteilt werden muss.__* Um das zu erreichen, haben wir das „The Things Network“ gegründet.
```

Das „The Things Network“ ist eine quelloffen orientierte und unabhängige Bewegung mit den folgenden Grundsätzen:

* Es verbindet Sensoren und Aktoren, welche als „Things“ bezeichnet werden, über „Things Gateways“  mit Computersystemen, die „Things Access“ genannt werden.

> * Der erste Teil einer solchen Verbindung findet drahtlos („Over The Air“) statt, der zweite Teil in einem Computernetzwerk („Over The Net“). Die dafür notwendigen Systeme, Konzepte und deren Aufbau nennen sich das „The Things Network“.

Reviewer: ```suggestion
* Der erste Teil einer solchen Verbindung findet drahtlos („Over The Air“) statt, der zweite Teil in einem Computernetzwerk („Over The Net“). Die dafür notwendigen Systeme, Konzepte und deren Aufbau nennen sich „The Things Network“.
```

> * Jeder ist frei eigene „Things“ zu betreiben und diese mit einem „Things Gateway“  zu verbinden, unabhängig von dessen Eigentumsverhältnis.

Reviewer: ```suggestion
* Jeder ist frei, eigene „Things“ zu betreiben und diese mit einem „Things Gateway“  zu verbinden, unabhängig von dessen Eigentumsverhältnis.
```

> * Jeder ist frei eigene „Things Gateways“ zu betreiben und diese mit einem „Things Access“ zu verbinden, unabhängig von dessen Eigentumsverhältnis. Diese „Things Gateways“ geben Zugriff zu allen „Things“ mit einem Ansatz der Netzneutralität alleinig limitiert durch die zur Verfügung stehende Bandbreite.

Reviewer: ```suggestion
* Jeder ist frei eigene „Things Gateways“ zu betreiben und diese mit einem „Things Access“ zu verbinden, unabhängig von dessen finanziellen Möglichkeiten. Diese „Things Gateways“ geben Zugriff zu allen „Things“ mit einem Ansatz der Netzneutralität, lediglich limitiert durch die zur Verfügung stehende Bandbreite.
```

> * Jeder ist frei einen eigenen „Things Access“ zu betreiben und anonyme Verbindungen über das Internet bereitzustellen. Dieses „Things Access“ erlaubt Zugriff auf alle „Things Gateways“ mit einem Ansatz der Netzneutralität alleinig limitiert durch die zur Verfügung stehende Bandbreite. Dieses „Things Access“ erlaubt auch die Verbindung mit anderen „Things Access“, um Daten auszutauschen.

Reviewer: ```suggestion
* Jeder ist frei einen eigenen „Things Access“ zu betreiben und anonyme Verbindungen über das Internet bereitzustellen. Dieses „Things Access“ erlaubt Zugriff auf alle „Things Gateways“ mit einem Ansatz der Netzneutralität, lediglich limitiert durch die zur Verfügung stehende Bandbreite. Dieses „Things Access“ erlaubt auch die Verbindung mit anderen „Things Access“, um Daten auszutauschen.
```

* Die Verbindungsteile „Over The Air“ und „Over The Net“ sollen derart ausgestaltet sein, dass diese unabhängig von einem spezifischen Protokoll sind. Alle genutzten Protokolle sollen nicht proprietär, quelloffen und kostenfrei nutzbar sind.
* Jeder, der ein „Things Access“ oder „Things Gateway“ betreibt, wird diese kostenfrei für alle damit verbundenen Geräte und Server bereitstellen.

> * Jeder der das „The Things Network“ benutzt, darf dies für beliebige Zwecke tun, beschränkt lediglich durch lokale Rechtsvorschriften, auf eigenes Risiko und mit dem Wissen, dass die Bereitstellung ohne Gewährleistung erfolgt und jederzeit ohne Ankündigung und ohne Angabe von Gründen beendet werden kann. Produkte und Services, die sich aus der Nutzung ableiten dürfen Sie frei für jeden anbieten, kommerzielle Zielen verfolgen, nur bestimmten Personen zur Verfügung stellen, ohne Gewinnerzielungsabsicht oder mit jeder anderen Absicht anbieten. Anbieter des „The Things Network“ erlegen ihren Nutzern keine Beschränkungen auf.

Reviewer: ```suggestion
* Jeder der das „The Things Network“ benutzt, darf dies für beliebige Zwecke tun, beschränkt lediglich durch die jeweils gültigen lokalen Rechtsvorschriften, auf eigenes Risiko und mit dem Wissen, dass die Bereitstellung ohne Gewährleistung erfolgt und jederzeit ohne Ankündigung und ohne Angabe von Gründen beendet werden kann. Produkte und Services, die sich aus der Nutzung ableiten, dürfen Sie frei für jeden anbieten, kommerzielle Zielen verfolgen, nur bestimmten Personen zur Verfügung stellen, ohne Gewinnerzielungsabsicht oder mit jeder anderen Absicht anbieten. Anbieter des „The Things Network“ erlegen ihren Nutzern keine Beschränkungen auf.
```
